tdw_physics/util.py

```diff
@@ -49,18 +59,29 @@ def get_object_look_at(o_id: int, pos: Dict[str, float], noise: float = 0) -> Li
     return commands
 
 
-def get_args(dataset_dir: str):
+def get_parser(dataset_dir: str, get_help: bool=False):
     """
     :param dataset_dir: The default name of the dataset.
 
     :return: Parsed command-line arguments common to all controllers.
     """
 
-    from argparse import ArgumentParser
-    parser = ArgumentParser()
+    import argparse
+    parser = argparse.ArgumentParser(add_help=get_help)
     parser.add_argument("--dir", type=str, default=f"D:/{dataset_dir}", help="Root output directory.")
-    parser.add_argument("--num", type=int, default=3000, help="The number of trials in the dataset.")
+    parser.add_argument("--num", type=int, default=3, help="The number of trials in the dataset.")
     parser.add_argument("--temp", type=str, default="D:/temp.hdf5", help="Temp path for incomplete files.")
     parser.add_argument("--width", type=int, default=256, help="Screen width in pixels.")
     parser.add_argument("--height", type=int, default=256, help="Screen width in pixels.")
+    parser.add_argument("--seed", type=int, default=0, help="Random seed with which to initialize scenario")
+    parser.add_argument("--random", type=int, default=1, help="Whether to set trials randomly")
+    parser.add_argument("--num_views", type=int, default=1, help="How many possible viewpoints to render trial from")
+    parser.add_argument("--viewpoint", type=int, default=0, help="which viewpoint to render from")

```

Comment:
Remove `--num_view` and the other args specific to `stability.py` and move them to `PrimitivesDataset`. For an example of controller-specific args, see `table_scripted.py`